content/blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Kestra Raises $3 Million to Build the Universal Open Source Orchestration Solution for All Engineers"
```

Comment:
```suggestion
title: "Kestra Raises $3 Million to Simplify and Unify Orchestration for All Engineers"
```

to build makes it sound as we start to build, the product already exists so maybe this